Sonify/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "sonify",
+  "version": "0.3.2",
+  "description": "A CODAP Plugin that plays temporal data as sounds.",
+  "private": true,
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
+    "test": "echo \"No automated tests at this time. See test/test-cases.md.\" && exit 1"
+  },
+  "author": "Takahiko Tsuchiya",
+  "license": "ISC",
+  "dependencies": {
+    "nexusui": "^2.1.6",
```

Comment:
If I remember correctly, the NexusUI used in the plugin might be v1 (need to check). v1 -> v2 was a complete rewrite and had many breaking changes.

Reply:
I looked at NexusUI.js in the old code-base and could not find a version number. I haven't seen any problems with the version we are now using, so maybe the incompatibilities are in unused parts. The only problems I have seen is that this library contributes a good deal to the size of the uploaded module.